core/src/main/java/global/maplink/helpers/MapHelpers.java

```diff
@@ -0,0 +1,23 @@
+package global.maplink.helpers;
+
+import lombok.NoArgsConstructor;
+import lombok.val;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@NoArgsConstructor(access = PRIVATE)
+public class MapHelpers {
+
+    public static Map<String, String> mapOf(String... values) {
```

Comment:
No java11, tem o `Map.of`, não daria para usar ele em vez desse método ?

Reply:
Mas se o cara usar o SDK em um projeto com Java 8, nao pode acabar dando conflito?

Reply:
É verdade, esqueci desse detalhe que o SDK está como java8.